tests/fixtures/noqa.py

```diff
@@ -593,3 +593,19 @@ def nested_annotation_return_func() -> Union[str, Union[int, float]]: # noqa: WP
 print(constant[0:7])  # noqa: WPS349
 
 var_a = var_a + var_b  # noqa: WPS350
+
+class ParentClass(object):
```

Comment:
Can you please remove this class? It works fine, and in `test_noqa` we only test that `# noqa` comments work. So, there's no need in correct examples. We only care about incorrect ones.

Which is already present.